settings/Core.setting.php

```diff
@@ -961,28 +966,28 @@
     'group' => 'core',
     'name' => 'syncCMSEmail',
     'type' => 'Boolean',
+    'html_type' => 'YesNo',
     'quick_form_type' => 'YesNo',
     'default' => 1,
     'add' => '4.7',
-    'title' => 'Sync CMS Email',
+    'title' => ts('Sync CMS Email'),
     'is_domain' => 1,
     'is_contact' => 0,
-    'description' => 'If enabled, then CMS email id will be syncronised with CiviCRM contacts\'s primary email.',
+    'description' => ts('If enabled, then CMS email id will be synchronised with CiviCRM contacts\'s primary email.'),
     'help_text' => NULL,
   ),
   'preserve_activity_tab_filter' => array(
     'group_name' => 'CiviCRM Preferences',
     'group' => 'core',
     'name' => 'preserve_activity_tab_filter',
-    'type' => 'String',
-    'html_type' => 'Text',
+    'type' => 'Boolean',
+    'html_type' => 'checkbox',
     'default' => '0',
     'add' => '4.7',
-    'title' => 'Preserve activity filters as a user preference',
+    'title' => ts('Preserve activity filters as a user preference'),
```

Comment:
Title markup entries should probably end with a full stop like most of the description entries do.

Reply:
as discussed - leaving full stops off the titles